content/Mobile_Dev/WebAuthn/IOS/Register_Only_Face_ID.adoc

```diff
@@ -0,0 +1,129 @@
+
+Take a look at Figure 1 to see an example of an account security screen for adding a credential.
+
+image::images/reg_faceid_1.jpg[width=250]
+**Figure 1**
+
+Once the user is ready to register a new credential, the “Add Face ID” button is clicked.
+
+Figure 2 shows the modal that appears on the user's screen asking if they wish to use Face ID.
+
+image::images/reg_faceid_2.jpg[width=500]
+**Figure 2**
+
+This is different from the standard modal pop-ups. Previously the user would have been asked if they wanted to use Face ID, with an option to register a security key. Instead they are now only given the option to register using Face ID(an example of the default flow can be seen on link:/Mobile_Dev/WebAuthn/IOS/Register_Face_ID_and_Security_Keys.html[this page]).
+
+Behind the scenes, the button is triggering a call to your relying party in order to retrieve the PublicKeyCreationOptions that are used to create the new credential.
+
+Figure 3 demonstrates an example of the PublicKeyCreationOptions that were used in the ceremony above. 
+
+[source, json]
+----
+{
+  "publicKey": {
+      "rp": {
+          "name": "WebAuthn Starter Kit",
+          "id": "d2a4za4g31xyaw.amplifyapp.com/"
+      },
+      "user": {
+          "name": "ios-demo",
+          "displayName": "ios-demo",
+          "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+      },
+      "challenge": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+      "pubKeyCredParams": [***],
+      "excludeCredentials": [
+          {
+              "type": "public-key",
+              "id": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXXXXXXXXXXXXXXXXXXXX-XXXXXXXXXXXX"
+          }
+      ],
+      "authenticatorSelection": {
+          "authenticatorAttachment": "platform",
+          "residentKey": "preferred",
+          "userVerification": "preferred"
+      },
+      "attestation": "direct",
+      "extensions": {
+          "credProps": true
+      }
+  }
+}
+----
+**Figure 3**
+
+While this sample looks very similar to the options that allow for both Face ID and security keys, the primary difference can be noted in line 25. The authenticatorSelection object in the PublicKeyCreationnOptions contains a property titled authenticatorAttachment. 
+
+Setting authenticatorAttachment to platform will force the user to register their platform authenticator, in this case it's Face ID. This behavior will remove the initial prompt that included an option for security keys. Instead Safari's WebAuthn prompt will immediately ask a user to invoke Face ID, as seen in Figure 2.
+
+More information on the authenticatorAttachment can be found in link:https://www.w3.org/TR/webauthn-2/#enum-attachment[this section] of the WebAuthn specification.
```

Comment:
Consider renaming 'this section' to be more descriptive.

Reply:
Changed in the latest commit